src/OpenTelemetry.AutoInstrumentation/Configuration/IntegrationInfo.cs

```diff
@@ -20,18 +20,13 @@ namespace OpenTelemetry.AutoInstrumentation.Configuration;
 
 internal readonly struct IntegrationInfo
 {
-    public readonly string Name;
+    public readonly string? Name;
 
     public readonly int Id;
 
-    public IntegrationInfo(string integrationName)
+    public IntegrationInfo(string? integrationName)
```

Comment:
Seems a candidate for refactoring. There is no point to allow `integrationName` to be `null`.

Reply:
You are right. Still Name field has to be nullable - as we have 2 separate constructors.